basicview.cpp

```diff
@@ -18,38 +18,77 @@ limitations under the License.
 
 #include <QVBoxLayout>
 
+#include <QFontDatabase>
+#include <QFont>
+#include <QDebug>
+#include <QChar>
+
+const QString startIcon = "\uf04b";
+const QString stopIcon = "\uf04d";
+const QString pauseIcon = "\uf04c";
+const QString deleteIcon = "\uf1f8";
+const QString reloadIcon = "\uf021";
+
+void BasicView::setFA(QWidget *wid)
+    {
+    if (QFontDatabase::addApplicationFont(":/images/FontAwesome.otf") < 0)
```

Comment:
I think the font load should only occur once and not for every button. Maybe move this to the `window.cpp` file so it occurs on start once

Reply:
good catch that was in my TODO list that got forgottne